src/brainglobe_napari/atlas_viewer_widget.py

```diff
@@ -161,20 +167,61 @@ def _on_selection_changed():
                     self._model.index(self._selected_atlas_row, 0)
                 )
                 self.refresh_info_box()
+                self.refresh_structure_tree_view()
             else:
                 self.atlas_info.setText("")
 
         self.atlas_table_view.selectionModel().selectionChanged.connect(
             _on_selection_changed
         )
 
+        # show structures in a tree view
+        self.structure_tree_view = QTreeView()
+        self.structure_tree_view.hide()
+
+        self.add_structure_button = QPushButton()
+        self.add_structure_button.setText("Add structure mesh")
+        self.add_structure_button.hide()
+
+        def _on_add_structure_clicked():
+            """Links add structure button to selected row
+            in the structure tree view"""
+            selected_index = (
+                self.structure_tree_view.selectionModel().currentIndex()
+            )
+            if selected_index.isValid():
+                selected_structure_name = (
+                    self.structure_tree_view.model().data(selected_index)
+                )
+                selected_atlas = BrainGlobeAtlas(self._selected_atlas_name)
+                selected_atlas_representation = NapariAtlasRepresentation(
+                    selected_atlas, self._viewer
+                )
+                selected_atlas_representation.add_structure_to_viewer(
+                    selected_structure_name
+                )
+            else:
+                show_info(
+                    "No structure selected. Select a structure first \
+                    to add it to napari."
+                )
+
+        self.add_structure_button.clicked.connect(_on_add_structure_clicked)
+
         # add sub-widgets to top-level widget
         self.layout().addWidget(self.atlas_table_view)
         self.layout().addWidget(self.download_selected_atlas)
         self.layout().addWidget(self.add_to_viewer)
-        self.layout().addWidget(self.atlas_info)
+
+        atlas_info_collapsible = QCollapsible("Atlas info")
```

Comment:
Is it possible to customise the symbol in any way? It might just be me, but I see the `△` as "caution" not "please click on me".

Reply:
I'm not going to bother with addressing this because this will become a tooltip anyway after today's discussion, see #45